docs/RationaleRTL-SV.md

```diff
@@ -53,6 +53,59 @@ TODO: Describe inout types.  Analogy to lvalues vs rvalues.  Array indices for
 both forms.  Arrays, structs,
 moving [UnpackedArray](https://github.com/llvm/circt/issues/389) to SV someday.
 
+All types (excepting `UnpackedArray`) are packed, meaning no padding or
+alignment.
+
+### Integrals
+
+MLIR's `IntegerType` with `Signless` semantics are used to represent integral
+types (logic/wire/reg in SystemVerilog).
+
+- **Bit layout**: MSB -> LSB matches between logical and hardware.
+- **Padding**: None
+- **Alignment**: None
```

Comment:
The padding/alignment behavior is an artifact of the bitcast operation, so I'd recommend describing it in its own description in tablegen.  It is ok for the description to be long.

There you can just say "there is never padding/alignment"